### PR TITLE
feat: Update Spot illustrations to include new heart assets

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
# Objective

- Update the Spot.tsx file to include /heart/ subfolder for all the asset urls.
- Created new Spot components for all missing illustrations compared to UI kit.
- Updated Spot stories to include new illustrations.

# Motivation and Context
[KDS-17](https://cultureamp.atlassian.net/jira/software/projects/KDS/boards/289?selectedIssue=KDS-17)

### CODE REVIEW

File: draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx [hunk 1]
```diff
@@ -725,6 +841,167 @@ export const OneOnOne = ({ enableAspectRatio, ...props }: SpotProps) => (
...
+)
+export const Resources = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-resources.svg"
+  />
+)
+
+// TODO: Compare with Covid-19 Response spot
```
Comment (lloydstubber) [inline draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx:977]: Do you still need this?

File: draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx [hunk 2]
```diff
@@ -725,6 +841,167 @@ export const OneOnOne = ({ enableAspectRatio, ...props }: SpotProps) => (
...
+export const TempResponse = ({ enableAspectRatio, ...props }: SpotProps) => (
+  <Base
+    aspectRatio={enableAspectRatio ? "square" : undefined}
+    {...props}
+    name="illustrations/heart/spot/miscellaneous-response.svg"
+  />
+)
```
Comment (lloydstubber) [inline draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx:984]: Nit but should the variable and path be somewhat similar?

File: draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx (hunk 1 above)
Comment (NatalieLoyzaga) [reply draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx:977]: Good find - I will fix that up

Comment (NatalieLoyzaga) [reply draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx]: @lloydstubber - I have updated it to remove as it was a duplicate illustration.

**Review by lloydstubber — DISMISSED** [review]
LGTM!

**Review by lloydstubber — APPROVED** [review]

**Pull request merged**